components/brave_news/browser/brave_news_controller.h

```diff
@@ -154,6 +156,7 @@ class BraveNewsController : public KeyedService,
   PublishersController publishers_controller_;
   ChannelsController channels_controller_;
   FeedController feed_controller_;
+  FeedV2Builder feed_v2_builder_;
```

Comment:
Note: We're trying not to use the Controller terminology in Brave News anymore :laughing:

Reply:
Is it because `Controller` is too ambiguious?

Reply:
I think partly, but also because it already has a meaning in MVC land, and this isn't really that :laughing: